EMS/helpers/src/File/TempFile.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\Helpers\File;
+
+use Psr\Http\Message\StreamInterface;
+
+class TempFile
+{
+    public string $path;
+
+    private function __construct(string $name)
```

Comment:
To me the $name can be null and then initialized with tempnam()

Reply:
Fixed, but for the api -> file -> download, we pass the hash to prevent create a temp file every time. Aggree?